pkg/genericapiserver/genericapiserver.go

```diff
@@ -128,6 +131,18 @@ type APIGroupVersionOverride struct {
 	ResourceOverrides map[string]bool
 }
 
+// Info about a group version.
+type GroupVersionInfo struct {
+	GroupMeta latest.GroupMeta
+	// Info about the resources in this group version.
+	ResourcesStorageMap map[string]rest.Storage
+	// True, if this is the legacy core api ("/api").
+	IsLegacyCoreGroupVersion bool
```

Comment:
I think we should reserve the keyword "core" since we might want to name a group "core" later. "legacy" alone is clear enough to me. WDYT?